syntax/ps1.vim

```diff
@@ -53,35 +55,47 @@ syn match ps1Constant +\$^+
 " Keywords reserved for future use
 syn keyword ps1Keyword class define from using var
 
-" Functions and Cmdlets
-syn match ps1Cmdlet /\w\+-\w\+/
```

Comment:
I pulled the recent changes and still don't see cmdlets highlighted (example, `split-path`). See https://github.com/heaths/profile/blob/master/Microsoft.PowerShell_profile.ps1 for my `$profile`. I'm comparing PProvost/master to yours by checking out either from my pathogen bundle submodule. Can't see how that would make a difference, but thought I'd mention it.